GameServerLib/Logic/Content/ContentFile.cs

```diff
@@ -50,21 +50,27 @@ public string GetString(string section, string name, string defaultValue = "")
             return obj == null ? defaultValue : obj;
         }
 
-        public int GetInt(string section, string name, int defaultValue = 0)
+        public float GetFloat(string section, string name, float defaultValue = 0)
         {
             var obj = GetObject(section, name);
-            return obj == null ? defaultValue : Int32.Parse(obj);
+            try
+            {
+                return obj == null ? defaultValue : float.Parse(obj, CultureInfo.InvariantCulture);
+            }
+            catch (FormatException formate)
+            {
+                return defaultValue;
+            }
```

Comment:
float value;
    if (!float.TryParse(GetObject(section, name), out value)) return defaultValue;
    return value;

Bit clearer to see whats going on, doesn't need the null check. If we ever move to C# 7, we can even do this:

    if (!float.TryParse(GetObject(section, name), out var value)) return defaultValue;
    return value;